### PR TITLE
Cross-mio updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
CruiseInformation CSV:
Updated CCGS Tully to CCGS John P. Tully to match the shiplist CSV

Vocab CSV:
Added 2 reference designators for Southern Ocean 5 cruise for the GS/CUSTARD mooring (extra CPM)

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -219,7 +219,7 @@ OC1901A,R/V Oceanus,2019-01-11T00:00:00,2019-01-15T00:00:00,Endurance opportunit
 SU20190125,R/V Pacific Surveyor,2019-01-25T16:30:00,2019-01-26T04:26:00,Endurance Glider Deployments
 WJ20190203,M/V Warren Jr.,2019-02-03T09:00:00,2019-02-04T21:00:00,"Pioneer 11a, OSSM D9 buoy recovery, GL388 D7 recovery"
 TS20190205,F/V Tenacious II,2019-02-05T00:30:00,2019-02-06T01:00:00,"Pioneer 11b, glider deployments"
-TY-1902,CCGS Tully,2019-02-05T12:00:00,2019-02-23T12:00:00,Papa Glider gp_276 recovery on the Line P 2019-002 cruise
+TY-1902,CCGS John P. Tully,2019-02-05T12:00:00,2019-02-23T12:00:00,Papa Glider gp_276 recovery on the Line P 2019-002 cruise
 WJ20190228,M/V Warren Jr.,2019-02-28T17:00:00,2019-03-02T13:00:00,"Pioneer 11c, AUV deployments, glider cp_339 recovery"
 SKQ201905S,R/V Sikuliaq,2019-03-24T00:00:00,2019-03-24T00:00:00,Endurance opportunistic glider ce_383 recovery
 EK20190403,R/V Elakha,2019-04-03T15:00:00,2019-04-03T22:00:00,Endurance ce01issp_00012 and ce02shsp_00012 deployments
```
Comment (snwhite) [inline cruise/CruiseInformation.csv:225]: Looks good.

File: vocab/vocab.csv [hunk 2]
```diff
@@ -1379,6 +1379,8 @@ GP05MOAS-PG576-04-FLORTO000,Global Station Papa,Mobile Assets,Profiling Glider 5
 GP05MOAS-PG576-05-NUTNRM000,Global Station Papa,Mobile Assets,Profiling Glider 576,Nitrate,Satlantic,SUNA V2,0,200
 GP05MOAS-PG576-06-PARADM000,Global Station Papa,Mobile Assets,Profiling Glider 576,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,200
 GS01SUMO,Global Southern Ocean,Apex Surface Mooring,,,,,0,4800
+GS01SUMO-RIC21,Global Southern Ocean,Apex Surface Mooring,Near Surface Instrument Frame,,,,12,12
+GS01SUMO-RIC21-00-CPMENG000,Global Southern Ocean,Apex Surface Mooring,Near Surface Instrument Frame,Platform Controller,WHOI,Communications and Power Manager,12,12
```
Comment (snwhite) [inline vocab/vocab.csv:1383]: This looks right, but I think these 2 lines should be inserted before GS01SUMO-RID27-00-DCLENG000 below.

**Review by snwhite — APPROVED** [review]
Look good.  I just have a nit-picky comment about the placement of the 2 new lines in vocab.

File: vocab/vocab.csv [hunk 3]
```diff
@@ -1389,6 +1389,8 @@ GS01SUMO-RID16-05-PCO2WB000,Global Southern Ocean,Apex Surface Mooring,Near Surf
 GS01SUMO-RID16-06-DOSTAD000,Global Southern Ocean,Apex Surface Mooring,Near Surface Instrument Frame,Dissolved Oxygen,Aanderaa,Optode 4831,12,12
 GS01SUMO-RID16-07-NUTNRB000,Global Southern Ocean,Apex Surface Mooring,Near Surface Instrument Frame,Nitrate,Satlantic,ISUS,12,12
 GS01SUMO-RID16-08-SPKIRB000,Global Southern Ocean,Apex Surface Mooring,Near Surface Instrument Frame,Spectral Irradiance,Satlantic,OCR507 ICSW,12,12
+GS01SUMO-RIC21,Global Southern Ocean,Apex Surface Mooring,Near Surface Instrument Frame,,,,12,12
+GS01SUMO-RIC21-00-CPMENG000,Global Southern Ocean,Apex Surface Mooring,Near Surface Instrument Frame,Platform Controller,WHOI,Communications and Power Manager,12,12
 GS01SUMO-RID27-00-DCLENG000,Global Southern Ocean,Apex Surface Mooring,Near Surface Instrument Frame,Data Concentrator Logger (DCL),WHOI,Data Concentrator Logger,12,12
```
Comment (snwhite) [inline vocab/vocab.csv:1394]: Looks good.

**Review by snwhite — APPROVED** [review]

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (spetillo) [reply cruise/CruiseInformation.csv:225]: Looks good.

File: vocab/vocab.csv (hunk 3 above)
Comment (spetillo) [reply vocab/vocab.csv:1394]: Looks good.

**Review by spetillo — APPROVED** [review]
Looking good. Thanks!!

**Pull request merged**